src/components/Menu/Sidebar.js

```diff
@@ -3,20 +3,29 @@ import MuiDrawer from '@mui/material/Drawer';
 import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
 import Divider from '@mui/material/Divider';
 import SideMenu from './SideMenu';
+import { drawerWidth } from '../../constants/constants';
 
-const drawerWidth = 240;
-
-const Sidebar = ({ open, setOpen, children, selectedLink }) => {
+const Sidebar = ({
+    isOpen: open,
+    handleDrawerClose,
+    handleDrawerOpen,
+    children,
+    selectedLink,
+}) => {
     return (
         <>
             <Drawer variant="permanent" open={open}>
                 <DrawerHeader>
-                    <IconButton onClick={() => setOpen(false)}>
+                    <IconButton onClick={() => handleDrawerClose()}>
```

Comment:
**From**
```jsx
<IconButton onClick={() => handleDrawerClose()}>
```

**To**
```jsx
<IconButton onClick={handleDrawerClose}>
```

이렇게 작성하셔도 잘 동작할 것 같습니다!

> I think it will work well even if you write it like this!